SPECS/ruby/ruby.spec

```diff
@@ -19,8 +19,6 @@
 %global csv_version             3.2.8
 %global date_version            3.3.4
 %global delegate_version        0.3.1
-%global did_you_mean_version    1.6.3
```

Comment:
why these removes?

Reply:
I was looking at the page https://github.com/ruby/ruby/blob/v3_3_5/NEWS.md#stdlib-updates. 
I thought that the 3.3.5 version of Ruby does not package these default gems.
However, I could be wrong, so will revert.

Reply:
Reverted.